docs/installation/linux.md

```diff
@@ -2,7 +2,7 @@
 
 Supports [Ubuntu 14.04][u14], [Ubuntu 16.04][u16], [Ubuntu 17.04][u17], [Debian 8][deb8], [Debian 9][deb9],
 [CentOS 7][cos], [Red Hat Enterprise Linux (RHEL) 7][rhel7], [OpenSUSE 42.2][opensuse], [Fedora 25][fed25],
-[Fedora 26][fed26], and [Arch Linux][arch].
+[Fedora 26][fed26], [Arch Linux][arch], [Kali Linux][kali] and [Raspbian][rasp].
```

Comment:
Should we remove unsupported (EOL) distributives (Ubuntu 17.04, OpenSUSE 42.2, Fedora 25)?

Reply:
I think we should update the content about `Ubuntu 17.04, OpenSUSE 42.2, Fedora 25` during the preview.3 release. Now we still have old packages (17.04 and etc.) in the existing release pages, but in preivew.3, we will release `17.10` package.

@rjmholt Did you make any packigng changes for the RPM package? (anything we nee to do to support Fedora 27?)

Reply:
No updates to the RPM -- it all worked fine as is

Reply:
Open #6867 to track this work during the next release.

Reply:
The other thought I had when looking at new release packaging was the the URLs we use for Ubuntu packages should be updated.

Reply:
@rjmholt what URLs? Can you please update #6867 with that information?